cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/ReportService.java

```diff
@@ -47,18 +51,29 @@ public ReportService() {
     @GET
     @Path("/parameters/{report_type}")
```

Comment:
ReportService returns next info:
`{"sender":"no-replay@codenvy.com",
"title":"Weekly On-Prem report",
"receiver":"data@codenvy.com"}`
Did you mean this info is a sensitive?